test/Conversion/ExportVerilog/line-length.mlir

```diff
@@ -12,26 +14,37 @@ hw.module @longvariadic(%a: i8) -> (b: i8) {
+// SHORT:             + a + a + a + a + a + a + a + a + a + a +
+// SHORT:             a + a + a + a + a + a + a + a + a + a + a
+// SHORT:             + a + a + a + a + a + a + a + a + a + a +
+// SHORT:             a + a + a + a + a + a + a + a + a + a + a
+// SHORT:             + a + a + a + a + a + a + a + a + a + a +
+// SHORT:             a;
 
 // DEFAULT-LABEL: module longvariadic
-// DEFAULT: wire [7:0] _tmp = a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a;
-// DEFAULT: wire [7:0] _tmp_0 = a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a;
-// DEFAULT: wire [7:0] _tmp_1 = a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a;
-// DEFAULT: wire [7:0] _tmp_2 = a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a;
-// DEFAULT: assign b = _tmp + _tmp_0 + _tmp_1 + _tmp_2;
+// DEFAULT: assign b = a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a +
+// DEFAULT:            a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a +
+// DEFAULT:            a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a; 
 
 // LONG-LABEL: module longvariadic
-// LONG: wire [7:0] _tmp = a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a;
-// LONG: wire [7:0] _tmp_0 = a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a;
-// LONG: assign b = _tmp + _tmp_0;
+// LONG: assign b = a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a
+//                  + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a;
+
+// LIMIT_SHORT-LABEL: module longvariadic
+// LIMIT_SHORT:  wire [7:0] _tmp = a + a + a + a + a + a + a + a + a + a + a
+// LIMIT_SHORT:                    + a + a + a + a + a;
+// LIMIT_SHORT:  wire [7:0] _tmp_0 = a + a + a + a + a + a + a + a + a + a + a
+// LIMIT_SHORT:                      + a + a + a + a + a;
+// LIMIT_SHORT:  wire [7:0] _tmp_1 = a + a + a + a + a + a + a + a + a + a + a
+// LIMIT_SHORT:                      + a + a + a + a + a;
+// LIMIT_SHORT:  wire [7:0] _tmp_2 = a + a + a + a + a + a + a + a + a + a + a
+// LIMIT_SHORT:                      + a + a + a + a + a;
+// LIMIT_SHORT:  assign b = _tmp + _tmp_0 + _tmp_1 + _tmp_2;
+
+// LIMIT_-LABEL: module longvariadic
+// LIMIT_LONG:   wire [7:0] _tmp = a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a +
+// LIMIT_LONG:                     a + a + a + a + a + a + a + a + a;
+// LIMIT_LONG:   wire [7:0] _tmp_0 = a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a +
+// LIMIT_LONG:                       a + a + a + a + a + a + a + a + a;
+// LIMIT_LONG:   assign b = _tmp + _tmp_0;
```

Comment:
These tests look great! I really like the checking of both the short and long limits. Can you use `*-NEXT` here, too.